fedora/jellyfin.spec

```diff
@@ -26,14 +24,7 @@ Source16:       jellyfin-firewalld.xml
 %{?systemd_requires}
 BuildRequires:  systemd
 Requires(pre):  shadow-utils
-BuildRequires:  libcurl-devel, fontconfig-devel, freetype-devel, openssl-devel, glibc-devel, libicu-devel, git
-%if 0%{?fedora}
-BuildRequires:  nodejs-yarn, git
-%else
-# Requirements not packaged in main repos
-# From https://rpm.nodesource.com/pub_10.x/el/7/x86_64/
-BuildRequires:  nodejs >= 10 yarn
-%endif
```

Comment:
Nice to see this go.  But how are the web bits deployed with this change?

Reply:
Web will be a separate package (optional) with it's own build process in the other repository. We have a pull request up right now that will technically allow people to run the server without a bundled web client. The main goal is to improve support for headless installations and hopefully allow setup without requiring the web client eventually.

Reply:
Well, technically it will be recommended since most users will still want the web client installed automatically, but the general idea still stands.

Reply:
Yep, there will be two packages: `jellyfin-server` (this one), and `jellyfin-web`, plus a metapackage to install both called `jellyfin`.

Reply:
Fair enough.

Hrm.  In other GH projects I have a _Resolve Conversation_ button to help me track which items have been resolved and which have not.  Don't see that here.  :-(

Reply:
You might need to be an org member to do that, but I've done so, thanks!